pillar/edx/ansible_vars/next_residential.sls

```diff
@@ -39,3 +39,5 @@ edx:
         - name: git+https://github.com/Stanford-Online/xblock-in-video-quiz@release/v0.1.7#egg=xblock-in-video-quiz
           extra_args: -e
         - name: xblock-image-modal==0.4.2
+        # Python client for Sentry
+        - name: raven
```

Comment:
Should we add this to the cloud_deployments pillar?

Reply:
I didn't add it there as I didn't want to change how we're deploying the additional apps and was thinking of sticking to the existing pattern

Reply:
Works for me.